docs/source/design.rst

```diff
@@ -0,0 +1,241 @@
+========
+ Design
+========
+
+When a Matplotlib :obj:`~matplotlib.artist.Artist` object in rendered via the `~matplotlib.artist.Artist.draw` method the following
+steps happen (in spirit but maybe not exactly in code):
+
+1. get the data
+2. convert from unit-full to unit-less data
+3. convert the unit-less data from user-space to rendering-space
+4. call the backend rendering functions
+
+..
+   If we were to call these steps :math:`f_1` through :math:`f_4` this can be expressed as (taking
+   great liberties with the mathematical notation):
+
+   .. math::
+
+      R = f_4(f_3(f_2(f_1())))
```

Comment:
That the nu / Q / tau notation is not published yet so there is no reference and I do not want to introduce it here.  

Given this is meant to be math-ish not program-ish single letter functions are more natural and numbering them makes clear there is a sequence rather than 4 different letters.